k8s/examples/wordpress/wordpress/10-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress
+  labels:
+    app: wordpress
+    tier: frontend
+spec:
+  externalTrafficPolicy: Local
+  ports:
+    - port: 8080
```

Comment:
We shouldn't change the port. It is an anti pattern. Kubernetes solves the port mapping elegantly so it is recommended to use default ports for simplicity. It is definitely not wrong to do so, but...